tenant/tenantmiddleware.go

```diff
@@ -61,7 +61,7 @@ const (
 // Adds systemBaseUri and tenantId to request context.
 // If the headers are not present the given defaultSystemBaseUri and tenant "0" are used.
 // The signatureSecretKey is specific for each App and is provided by the registration process for d.velop cloud.
-func AddToCtx(defaultSystemBaseUri string, signatureSecretKey []byte) func(http.Handler) http.Handler {
+func AddToCtx(defaultSystemBaseUri string, signatureSecretKey []byte, logger func(ctx context.Context, message string)) func(http.Handler) http.Handler {
```

Comment:
Please use logError instead of logger for the name of the log func param to use a naming similar to the idp middleware.